### DISCUSSION TITLE
p/%E7%BA%BF%E4%B8%8A%E9%97%AE%E9%A2%98%E6%8E%92%E6%9F%A5%E6%96%B9%E6%B3%95%E6%B1%87%E6%80%BB/

### DISCUSSION THREAD

**Discussion by giscus** [baixiaozhou/hexo-blog#2] [Announcements]
# p/%E7%BA%BF%E4%B8%8A%E9%97%AE%E9%A2%98%E6%8E%92%E6%9F%A5%E6%96%B9%E6%B3%95%E6%B1%87%E6%80%BB/

线上故障问题的排查方法和工具介绍，包括 CPU、内存、负载、磁盘、IO、网络等

https://baixiaozhou.github.io/2024/08/15/%E7%BA%BF%E4%B8%8A%E9%97%AE%E9%A2%98%E6%8E%92%E6%9F%A5%E6%96%B9%E6%B3%95%E6%B1%87%E6%80%BB/

**Comment by baixiaozhou** [comment]
博客内容还在不断完善中，此评论用作测试使用

**Comment by panyy97** [comment]
匹夫，速更

**Comment by WangDX520** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀

**Comment by WangDX520** [comment]
彼之文也，笔力雄健。大哥佩服
